packages/react-core/src/components/TimePicker/TimePicker.tsx

```diff
@@ -48,6 +48,8 @@ export interface TimePickerProps
   direction?: 'up' | 'down';
   /** Size of step between time options in minutes.*/
   stepMinutes?: number;
+  /** Additional props for input field */
+  inputProps?: object;
```

Comment:
Try `TextInputProps`. You'll need to add `import { TextInputProps } from '../TextInput';` to the top of the file.